inference-engine/src/transformations/include/transformations/low_precision/subtract.hpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/ngraph.hpp>
+#include "transformations/low_precision/layer_transformation.hpp"
+
+namespace ngraph {
+namespace pass {
+namespace low_precision {
+
+class TRANSFORMATIONS_API SubtractTransformation : public LayerTransformation {
+public:
+    SubtractTransformation(const Params& params) : LayerTransformation(params) {}
+    ~SubtractTransformation() override {}
+    void registerMatcherIn(GraphRewrite& pass, TransformationContext& context) const override;
+    bool transform(TransformationContext& context, ngraph::pattern::Matcher &m) const override;
```

Comment:
That's an abstract class, `isPrecisionPreserved` is not implemented. 
There is no build failure only because it doesn't use... at all.